twilight-model/src/http/attachment.rs

```diff
@@ -21,18 +24,18 @@ use serde::{Deserialize, Serialize};
 /// .to_vec();
 /// let id = 1;
 ///
-/// let mut attachment = Attachment::from_bytes(filename, file_content, id);
+/// let mut attachment = Attachment::from_bytes(filename, Cow::from(file_content), id);
```

Comment:
Cow::from is not smart enough for `&'static [u8]` data (needed for it to work without `.to_vec()`)
```suggestion
/// let mut attachment = Attachment::from_bytes(filename, Cow::Borrowed(file_content), id);
```